katello/katello-service

```diff
@@ -73,6 +77,18 @@ def list_services
   puts `chkconfig --list 2>&1 | grep '#{regex}'`
 end
 
+def status_services(action)
+  if `which systemctl 2>&1` && $?.success? && !(@options.include?(:verbose))
+    regex = services_by_priority.map { |service| "^[^a-z_-]*#{service}.service" }.join('\|')
```

Comment:
The associated issue has a good example.  Basically it greps for each service, and the header line.  

```
[root@satellite6 bin]# ./katello-service-gh status
  UNIT                                                  LOAD      ACTIVE   SUB     DESCRIPTION
  foreman-proxy.service                                 loaded    active   running Foreman Proxy
  foreman-tasks.service                                 loaded    active   running Foreman jobs daemon
  goferd.service                                        loaded    active   running Gofer Agent
  httpd.service                                         loaded    active   running The Apache HTTP Server
  mongod.service                                        loaded    active   running High-performance, schema-free document-oriented dat                      abase
  postgresql.service                                    loaded    active   running PostgreSQL database server
  pulp_celerybeat.service                               loaded    active   running Pulp's Celerybeat
  pulp_resource_manager.service                         loaded    active   running Pulp Resource Manager
  pulp_streamer.service                                 loaded    active   running The Pulp lazy content loading streamer
  pulp_workers.service                                  loaded    active   exited  Pulp Celery Workers
  qdrouterd.service                                     loaded    active   running Qpid Dispatch router daemon
  qpidd.service                                         loaded    active   running An AMQP message broker daemon.
  smart_proxy_dynflow_core.service                      loaded    active   running Foreman smart proxy dynflow core service
  squid.service                                         loaded    active   running Squid caching proxy
  tomcat.service                                        loaded    active   running Apache Tomcat Web Application Container
```